enemies/enemy_1/enemy_1.gd

```diff
@@ -0,0 +1,16 @@
+extends Area2D
+
+const SPEED = 250.0
+const COLLISION_DAMAGE = 20
+
+func _physics_process(delta):
+	var player_location = Globals.player.global_position
+	var movement_vector = delta * SPEED
+	global_position = global_position.move_toward(player_location, movement_vector)
```

Comment:
same thing you were doing before, just using `move_toward`, I changed this from PlayerCharacter2D to Area2D (generic collision object). i'm not sure what the advantages of the character node are, maybe we switch back at some point